provider/cpu_utils.py

```diff
@@ -168,3 +169,35 @@ def check_if_vm_vcpus_match_qemu(vm):
     enabled_count = vm.cpuinfo.smp + (len(enabled_vcpu_devices) * vcpus_count)
 
     return check_if_vm_vcpu_match(enabled_count, vm)
+
+
+def check_hotplug_ovmf_support(params, test, plug=True, unplug=False):
```

Comment:
This function check both qemu and ovmf version, and include both hotplug and hotunplug, so the function name looks confusing for me.  How about `check_version_support`